exercises/input-output/src/main/java/com/cbfacademy/WriteStream.java

```diff
@@ -0,0 +1,35 @@
+package com.cbfacademy;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class WriteStream {
+    public void writeStream() throws IOException{
+        String filePath = "src/main/resources/exercise.txt";
+        // input filepath
+        String outputPath = "src/main/resources/writingExercise.txt";  
```

Comment:
If we pass these to the method as arguments instead of hardcoding them here, we can then reuse this method any time we want to read from one file and write to another.